multistateview/src/main/java/me/xdj/view/SimpleMultiStateView.java

```diff
@@ -0,0 +1,97 @@
+package me.xdj.view;
+
+import ohos.agp.components.AttrSet;
+import ohos.agp.components.Component;
+import ohos.app.Context;
+
+/**
+ * Created by xdj on 16/2/3.
+ * Multi-state view.
+ */
+public class SimpleMultiStateView extends MultiStateView implements Component.BindStateChangedListener {
+
+    private static final int MIN_SHOW_TIME = 600;
+    private static final int MIN_DELAY = 600;
+    Integer emptyId;
+    Integer resIdLoading;
+    Integer resIdFail;
+    private int mTargetState = -1;
+    private long mLoadingStartTime = -1;
+
+    private final Runnable mLoadingHide = () -> {
+        setViewState(mTargetState);
+        mLoadingStartTime = -1;
+        mTargetState = -1;
+    };
+
+    public SimpleMultiStateView(Context context) {
+        this(context, null);
+    }
+
+    public SimpleMultiStateView(Context context, AttrSet attrs) {
+        this(context, attrs, "");
+    }
+
+    /**
+     * SimpleMultiStateView.
+     *
+     * @param context context.
+     * @param attrs attr.
+     * @param defStyleAttr defStyle.
+     */
+    public SimpleMultiStateView(Context context, AttrSet attrs, String defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+
+        String emptyViewString = attrs.getAttr("msv_emptyView").get().getStringValue();
```

Comment:
replace emptyViewString, emptyViewString1, emptyViewString2 variable names with meaningful ones. You can retain the variables naming convention from the base library.